mcs/db_cluster_client.go

```diff
@@ -257,6 +276,16 @@ func (r commonClusterResult) extract() (*dbClusterResp, error) {
 	return nil, err
 }
 
+// extract is used to extract result into short response struct
+func (r shortClusterResult) extract() (*dbClusterShortResp, error) {
+	var c *dbClusterShortRespOpts
+	err := r.ExtractInto(&c)
+	if err == nil {
```

Comment:
i guess it's better to use reversed `if err != nil` or even `if err := r.Extract...; err !nil {` as it's used to in go

Reply:
Rewrote to `if err != nil`